### DISCUSSION TITLE
如何进行高帧率播放的配置？

### DISCUSSION THREAD

**Discussion by chenyifan98** [hooke007/mpv_PlayKit#373] [Q&A 问答]
大佬您好，看到您是中文区MPV大V，想来咨询一个相关问题！
目前有一个任务是进行1080p360hz的avi无损视频播放，但是经常播放时出现掉帧卡顿，大概3s40帧左右，不稳定。
我尝试了很多种设置，如hwdec=auto，增大缓存时间和容量，关闭所有滤镜，均效果不明显，目前最有用的配置是提高优先度。
想请教您有什么办法进行配置或者使用python-mpv之类的来特异性提高播放能力吗？
我是用i9-9920、2080ti、128g内存做32g虚拟硬盘做为平台使用mpv进行播放，查过任务管理器，应该没有硬件瓶颈
我是一名自动化系研究生，有一定的编程基础，如果方便的话可以进行付费咨询吗?

**Comment by hooke007** [comment]
> 看到您是中文区MPV大V

不，我不是，也不想是.

> 我是一名自动化系研究生，有一定的编程基础

既然如此你肯定比我要基础好（我的主业只和商科相关），为何不利用有编程基础的优势自己研究呢。
我也只是兴趣使然研究mpv，主要仅关注满足自己的需求。

题外话：
我之前选择了直接砸钱7950x+4080来解决了硬件壁垒，所以这就是提高播放性能的最有效的方法（

**Comment by snylonue** [comment]
mpv-player/mpv#11122

可能和这个 issue 相关
